lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context_and_proc.h

```diff
@@ -785,7 +785,7 @@ void amf_smf_context_cleanup_pdu_session(ue_m5gmm_context_s* ue_context);
 // PDU session related communication to gNB
 int pdu_session_resource_setup_request(
     ue_m5gmm_context_s* ue_context, amf_ue_ngap_id_t amf_ue_ngap_id,
-    std::shared_ptr<smf_context_t>);
+    std::shared_ptr<smf_context_t>, bstring);
```

Comment:
provide the parameter names

Reply:
addressed review comment, added parameter names